### PR TITLE
update workspace dependencies

### PR DESCRIPTION
**Pull request by anita-steiner**
Update all workspace (webpack, loader, ..) dependencies

### CODE REVIEW

File: generators/workspace/templates/package.tmpl.json [hunk 1]
```diff
@@ -34,34 +34,34 @@
...
     "typedoc": "~0.17.8",
     "typescript": "~3.9.7",
-    "url-loader": "~4.1.0",
-    "webpack": "~4.43.0",
-    "webpack-bundle-analyzer": "~3.8.0",
-    "webpack-cli": "~3.3.12",
-    "webpack-dev-server": "~3.10.3"
+    "url-loader": "~4.1.1",
+    "webpack": "~4.44.2",
+    "webpack-bundle-analyzer": "~3.9.1",
```
Comment (thinkh) [inline generators/workspace/templates/package.tmpl.json:63]:
```suggestion
    "webpack-bundle-analyzer": "~3.9.0",
```

There is no published version 3.9.1 which results in the following error:

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for webpack-bundle-analyzer@~3.9.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'ordino_develop'
npm ERR! notarget 
```

Last version published 3.x version is `3.9.0` (see https://www.npmjs.com/package/webpack-bundle-analyzer).

Comment (thinkh) [reply generators/workspace/templates/package.tmpl.json:63]: Could we even upgrade to the most current version 4?

File: generators/workspace/templates/package.tmpl.json [hunk 2]
```diff
@@ -34,34 +34,34 @@
...
+    "webpack-cli": "~4.2.0",
```
Comment (thinkh) [inline generators/workspace/templates/package.tmpl.json:64]:
```suggestion
    "webpack-cli": "~3.3.12",
```

This change causes the following error:

```
> tsc -w

[12:19:18 PM] Starting compilation in watch mode...

[12:19:18 PM] Starting compilation in watch mode...

internal/modules/cjs/loader.js:800
    throw err;
    ^

Error: Cannot find module 'webpack-cli/bin/config-yargs'
Require stack:
- /home/holger/w/ordino_develop/node_modules/webpack-dev-server/bin/webpack-dev-server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/holger/w/ordino_develop/node_modules/webpack-dev-server/bin/webpack-dev-server.js:65:1)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/holger/w/ordino_develop/node_modules/webpack-dev-server/bin/webpack-dev-server.js'
  ]
}
```

The last version of the 3.x branch is `3.3.12`.

**Review by thinkh — CHANGES_REQUESTED** [review]
I tested the change with the Ordino workspace and got some errors. After changing the versions as described below, it works again.

File: generators/workspace/templates/package.tmpl.json (hunk 1 above)
Comment (anita-steiner) [reply generators/workspace/templates/package.tmpl.json]: yes, sorry - that was my fault; 4.0 not really because of the acorn problem we had

**Review by thinkh — APPROVED** [review]

**Pull request merged**